website/conferences/views.py

```diff
@@ -28,6 +29,22 @@
 
 logger = logging.getLogger(__name__)
 
+# Setup Keen client
+keen.project_id = settings.KEEN_PROJECT_ID
+keen.write_key = settings.KEEN_WRITE_KEY
+
+
+def _dt_to_iso8601(value):
```

Comment:
@icereval FYI i copied the function from here: https://github.com/CenterForOpenScience/osf.io/blob/develop/api_tests/files/views/test_file_detail.py#L21 and changed it a bit. I would guess there is no helper.